avm/ptn/alz/empty/modules/policy-definitions/main.bicep

```diff
@@ -0,0 +1,34 @@
+metadata name = 'avm/ptn/alz/empty/policy-definitions'
+metadata description = 'Azure Landing Zones - Bicep - Empty - Policy Definitions Module'
+
+targetScope = 'managementGroup'
+
+@description('Optional. Policy definitions to create on the management group.')
+param managementGroupCustomPolicyDefinitions policyDefinitionType[]?
```

Comment:
Just a sidenote (hence feel free to resolve): This is very strongly discouraged, at least in `res` modules.

We discussed this a while back and the strong suggestion (also by the PG) was to not wrap entire resources into objects/types but expose the 1st (name/tags) and 2nd (properties-description, properties-displayname) level parameters as actual parameters with the option for defaults etc. and only have types for evey further level. 

As this is a pattern and I cannot find a spec that would ask you to not do this (suprisingly so, cc: @eriqua - could swear there was one), I won't block the PR over it and just wanted to raise awareness.

Reply:
This is done for a good reason here. The reason is i just needed to change scopes to the target MG the PTN module creates and then create X amount of policy definitions quickly at that scope. Importantly by not creating more ARM deployments like the alternative approach would mean. 

This is so that this module runs in minutes and not hours for ALZ for policy defintion creation.

Best way to think of this approach is just becuase i need to change deployment scope, thats all.